pkg/util/testing/wrappers.go

```diff
@@ -499,3 +511,55 @@ func (rc *RuntimeClassWrapper) PodOverhead(resources corev1.ResourceList) *Runti
 func (rc *RuntimeClassWrapper) Obj() *nodev1.RuntimeClass {
 	return &rc.RuntimeClass
 }
+
+type LimitRangeWrapper struct{ corev1.LimitRange }
+
+func MakeLimitRange(name, namespace string) *LimitRangeWrapper {
+	return &LimitRangeWrapper{
+		LimitRange: corev1.LimitRange{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      name,
+				Namespace: namespace,
+			},
+			Spec: corev1.LimitRangeSpec{
+				Limits: []corev1.LimitRangeItem{
+					{
+						Type:                 corev1.LimitTypeContainer,
+						Max:                  corev1.ResourceList{},
+						Min:                  corev1.ResourceList{},
+						Default:              corev1.ResourceList{},
+						DefaultRequest:       corev1.ResourceList{},
+						MaxLimitRequestRatio: corev1.ResourceList{},
```

Comment:
you could remove these lines.

Reply:
yes, but then you need to check / alocate the empty map for every value set

Reply:
ahhh, sorry, I forgot these were maps :)